util/include/string.hpp

```diff
@@ -104,5 +104,9 @@ std::string toString(const Container& container, const char* const separator = "
   return out.str();
 }
 
+inline bool isValidHexString(const std::string& str) {
```

Comment:
Do you sure we need this? If the e signature is not a valid hex, then verification will fail in any case, right?

Reply:
What you're saying does not sound very security oriented.
Regardless, I did not invent the sanity check but rather inherited it :)